README.md

```diff
@@ -1,18 +1,52 @@
-# SurPath Hub Website
-This repository is dedicated to our community's website. Contributions are welcome to make our website more community-friendly.
+<h1>SurPath Hub</h1>
 
-![image](https://user-images.githubusercontent.com/73097560/117122166-2aea6180-adc8-11eb-863f-a92de0d96af7.png)
+The SurPath Hub is an <h3>open source community</h3>
+The aim of Surpath Hub is to every Asia student,career shifter or aspiring developer into an open source environment regardless of their position in life
```

Comment:
The aim of Surpath Hub is to every Asia student,career shifter or aspiring developer into an open source environment regardless of their position in life
```suggestion
The aim of Surpath Hub is to every Filipino student,career shifter or aspiring developer into an open source environment regardless of their position in life
```